configs/9.0/packages/expat/sources

```diff
@@ -40,12 +42,45 @@ ATSRC_PACKAGE_BUNDLE=main_toolchain
 
 atsrc_get_patches ()
 {
-        at_get_patch \
-	https://hg.mozilla.org/releases/mozilla-esr31/raw-diff/2f3e78643f5c/parser/expat/lib/xmlparse.c \
-	978566c4144504d170869619661fbd7d || return ${?}
+	# Delete Makefile that generates man
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/a6bb3358ddae.patch \
+		bc61e4806ae531941b652bb56abb3249 || return ${?}
+
+	# Added man pages
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/f1a8200593de.patch \
+		0c71eb2bb1b6750a96c3c5181f67de3e || return ${?}
+
+	#Fix CVE-2012-6702 and CVE-2016-5300
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/95dbdf173f19.patch \
+		ea1a061d504d47aa726d09dc639db1fa || return ${?}
```

Comment:
On Debian, this patch is supposed to be applied on top of 2.1.0.  Do you know if this has the desired effect on 2.1.1?

Reply:
@gftg85 Yes it has. Those fixes are related to the poor entropy that was used on libexpat. Those fixes was available upstream only to versions 2.2.0 or higher.